modules/nf-core/controlfreec/makegraph/main.nf

```diff
@@ -8,7 +8,7 @@ process CONTROLFREEC_MAKEGRAPH {
         'biocontainers/control-freec:11.6--h1b792b2_1' }"
 
     input:
-    tuple val(meta), path(ratio), path(baf)
+    tuple val(meta), path(ratio), path(baf), val(ploidy)
```

Comment:
this requires the ploidy param to be passed in

Reply:
I think I may have actually added, for #980 which turned out to be way more complicated and dropped on the priority list. Could also patch here for now to keep the old behavior and then fix there for good

Reply:
Should I define `ploidy` like this
```
ploidy = params.cf_ploidy && !params.cf_ploidy.toString().contains(",") ? [params.cf_ploidy] : params.cf_ploidy.split(',')
```
in `subworkflows/local/bam_variant_calling_somatic_all/main.nf` like you did in #980?

If you feel like it, you can also commit the required changes from #980 on "my" branch `update_modules`.

Reply:
980 is not at all mature and requires a bit more digging into it. not comfortable committing it as is without more testing. I remember that what you show above also had some issue. in any case it is too long too just add

Reply:
So I see two solutions for now

1. We don't update controlfreec
2. We set the ploidy parameter to `[]` (or something like) in the call to `MAKEGRAPH`

Reply:
I vote for `1 don't update`